vmdb/app/helpers/application_helper.rb

```diff
@@ -2921,5 +2921,13 @@ def breadcrumb_prohibited_for_action?
     !%w(accordion_select tree_select).include?(action_name)
   end
 
+  def my_server_id
+    MiqServer.my_server(true).id
```

Comment:
I think the `true` will force a reload from the database on every invocation.  Maybe it should be memo-ized:
```ruby
def my_server_id
  @my_server_id ||= MiqServer.my_server(true).id
end
```